### DISCUSSION TITLE
Vue not in development mode

### DISCUSSION THREAD

**Discussion by BSC-Spotter** [nuxt/nuxt#24982] [Questions (Nuxt 2)]
Hello
For some reason when using nuxt build and then running it doesn't put Vue.JS in production mode; causing the following message to be displayed in the console:
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html

Is this a bug? I thought build would automatically put it in production mode

**Comment by hansalves** [comment]
I'm having the same issue. It's weird since we're using nuxt 3 and vue 3, but that link refers to vue 2 documentation.

**Reply by TheAlexLichter** [reply]
@hansalves Do you have `debug: true` somewhere in your nuxt config?

(A reproduction would be helpful, hard to guess otherwise 🙈 )
